api/src/main/java/org/hawkular/alerts/api/model/trigger/Trigger.java

```diff
@@ -630,4 +665,7 @@ public String toString() {
                 + ", mode=" + mode + ", tags=" + tags + "]";
     }
 
+    public enum TriggerLifecycle {
+        ALERT_GENERATE, AUTO_DISABLE, AUTO_RESOLVE, DISABLE, ENABLE, CREATED, MODIFIED
```

Comment:
Should those all in present or all in past?

ENABLE vs createD & modifieD

Reply:
Yeah, should be in the present form (as they happen).